src/main/java/com/dbteam/controller/ability/StartHandler.java

```diff
@@ -19,7 +19,7 @@
 public class StartHandler implements CommandHandler{
 
     private static final String MSG_START =
-            "Hello! I am MoneySplittingBot. I can remember all your group expenses. ";
+            "Hello! I am MoneySplittingBot. I can remember all your group expenses.";
```

Comment:
Actually that space(' ') is necessary because more text is appended to the message that uses MSG_START (e.g. MSG_NO_GROUPS